seqr/views/apis/report_api.py

```diff
@@ -759,6 +781,9 @@ def _get_saved_discovery_variants_by_family(variant_filter, parse_json=False):
         if parse_json:
             parsed_variant = variant_by_guid[saved_variant.guid]
             parsed_variant['discovery_tag_names'] = {vt.variant_tag_type.name for vt in saved_variant.discovery_tags}
+            parsed_variant['discovery_tag_guids_by_name'] = {
```

Comment:
so thinking about this more, this is actually some unneccessary duplication. Although its a slightly larger code change, I think we should actually replace `discovery_tag_names` with `discovery_tag_guids_by_name`. We would create it more cleanly than you do here, by doing
```
parsed_variant['discovery_tag_guids_by_name'] = {vt.variant_tag_type.name: vt.guid for vt in saved_variant.discovery_tags}
```
and then everywhere that currently uses `variant['discovery_tag_names']` would instead use `variant['discovery_tag_guids_by_name'].keys()`